package.json

```diff
@@ -19,6 +19,7 @@
     "colors": "~1.3.0",
     "compression": "~1.7.0",
     "cookie-parser": "~1.4.0",
+    "dotgitignore": "~1.0.3",
```

Comment:
adding a whole dependency to split a file's contents by newlines isn't really necessary

Reply:
this dependency is also doing the checking for us whether or not a file is affected by the .gitignore, returning true or false whether or not the file you pass is it meant to be ignored by gitignore. this saves us having to do a lot of things besides just splitting a file into new lines

Reply:
that's not hard to do manually though and the dependency isn't a terribly active project. we've run into problems in the past with adding dependencies that aren't being actively maintained and were never much code to begin with. the rule of thumb is a dependency needs to have some decent level of substance and active maintenance behind it before it's worth including in a project, and this one isn't quite passing the sniff test